### ISSUE TITLE
gene symbols for alevin-fry

### ISSUE THREAD

**Issue by grst** [nf-core/scrnaseq#184]
### Description of feature

Folow up of #160 by @kafkasl. We don't yet add gene symbols in the output files when using alevin-fry as aligner. 

> As per [this discussion on slack](https://nfcore.slack.com/archives/CHN5BV5DW/p1668433349777579) we now know a way to get the gene mapping for alevin (-> upgrade simpleaf version). But we agreed on following up on that in a separate PR.

**Comment by grst** [comment]
Depends on #361

**Comment by grst** [comment]
Should by now be consistently added to the h5ad and rds outputs by the matrix conversion.
